### ISSUE TITLE
Change book loader to remove context error message?

### ISSUE THREAD

**Issue by kmharrington** [simonsobs/sotodlib#618]
Even since we started loading with the smurf/detset ManifestDbs we're now getting this error on the AxisManager load: 
```
ERROR: sotodlib.core.context: Key "smurf" is present in det_info returned by observation loader as well as in metadata databases; The two versions are not comparable. dropping the loader version.
```

This really isn't an error, it's that we wanted that smurf key when we had no other information. I think we can just not load those things from the books now?

**Comment by mhasself** [comment]
> This really isn't an error, it's that we wanted that smurf key when we had no other information. I think we can just not load those things from the books now?

Sounds good... just so I'm clear, we can remove this (and everything else that make it possible)?

Perhaps we leave the code there, but disable it when using `load_obs_book` but enable it when using `load_book_file`?

**Comment by kmharrington** [comment]
I like the idea of disabling when loading from context and enabling when loading from `load_book_file`

**Comment by mhasself** [comment]
Resolved in #629
